azavea/cac-tripplanner
# Feature/kak/travel distances not times#1009

## Overview

Instead of finding the travel time to each destination and displaying it, show simple distance instead.

Add `turf.js` distance library to support calculating distances client-side, to take advantage of the destinations cache.

Also slightly modified line height of the distance field on the place card, to fix the bottom of some letters getting cut off.


### Demo

![distances_homepage](https://user-images.githubusercontent.com/960264/37224031-48ef5cf4-23a0-11e8-867a-e17dd32cfa2a.png)

Short distance formatting:
![formatted_short_distance](https://user-images.githubusercontent.com/960264/37224043-51970af0-23a0-11e8-823e-e6282f5b70bc.png)

Exactly one mile formats as singular:
![formatting_one_mile_exactly](https://user-images.githubusercontent.com/960264/37224063-5c586812-23a0-11e8-9971-7cbf60198909.png)



## Testing Instructions

 * Place cards should display no distance when origin not set
 * On setting origin, distances should display on place cards
 * Dragging to reroute should still work (check existing turf function)

Closes #1009
